vaadin-checkbox-flow-parent/vaadin-checkbox-flow/src/test/java/com/vaadin/flow/component/checkbox/tests/validation/CheckboxGroupBasicValidationTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.checkbox.tests.validation;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.vaadin.flow.component.checkbox.CheckboxGroup;
+import com.vaadin.tests.validation.HasValidationTestHelper;
+
+public class CheckboxGroupBasicValidationTest {
+    private CheckboxGroup<String> testField;
+
+    @Before
+    public void setup() {
+        testField = new CheckboxGroup<>();
+    }
+
+    @Test
+    public void setErrorMessage_getErrorMessage() {
+        HasValidationTestHelper.setErrorMessage_getErrorMessage(testField);
```

Comment:
Extracting the logic definitely makes sense, however this specific pattern results in a lot of noise in Sonar validation, because it does not detect the assertions:
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=4356&id=vaadin_flow-components&open=AYUQ_8ljB57jh_TraguY

Could we use an abstract test class instead, like `AbstractionBasicValidationTest` was used in the text field module?